Makefile

```
@@ -62,3 +71,14 @@ build: clean
 .PHONY: clean
 clean:
 	rm -rf dist
+
+
+# full hard reset
+.PHONY: nuke
+nuke:: local-uninstall clean
+	rm -rf *.egg-info || : ;
+	rm -vfRd ./**/__pycache__ 2>/dev/null || : ;
```

I do not think that we should delete all valid `__pycache__` files of the virtual environment as well.

---

I disagree, as the cache deletion in the venv is intentional [for testing](https://stackoverflow.com/a/28365204) here; otherwise developers need to manually reset the cache to ensure they are testing their changes. Further, I hit this very issue while working on GHI #2 (PR #54) when refactoring the code and then testing with the then stale bytecode, leading to import issues I had just fixed still occurring. This mundane cycle eventually resulting in me automating this very task here.

Ironically, I seem to have missed a cache even, let's consider:
```suggestion
	rm -vfRd ./**/__pycache__ 2>/dev/null || : ;
	rm -vfRd ./.mypy_cache 2>/dev/null || : ;
```

---

There are two aspects here:

* Clearing the cache of the module under development.
* Clearing the cache of other modules, for example of dependencies.

If I understand the command correctly, your command basically clears all `__pycache__` directories inside the virtual environment, including the dependencies - or am I mistaken? If I am correct, this does not make sense, as development should not do any write operations on dependencies.